eprihlaska/forms.py

```diff
@@ -227,12 +227,21 @@ class FurtherStudyInfoForm(FlaskForm):
     will_take_bio_matura = BooleanField(label=c.WILL_TAKE_BIO_MATURA)
 
 class FurtherGradesInfoForm(FlaskForm):
+    g_min = 1
+    g_max = 5
+    msg = 'Zle zadaná známka'
```

Comment:
@JakubNvk Toto prosim ta presun to `consts.py` a obal `_()` wrapperom, aby sa to potom mohlo prelozit.

Reply:
@mrshu len `msg` alebo aj `g_min` a `g_max`?

Reply:
@JakubNvk staci `msg`, ostatne asi nebudeme prekladat (ak som spravne pochopil, budu sa uvazovat iba slovenske znamky).